File: src/tests/ngraph_helpers/ngraph_functions/src/concat.cpp
// Copyright (C) 2018-2023 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//

#include <memory>
#include <vector>

#include "ngraph_functions/builders.hpp"
#include "openvino/op/concat.hpp"

namespace ngraph {
namespace builder {

std::shared_ptr<ov::Node> makeConcat(const std::vector<ov::Output<Node>>& in, const int& axis) {
    return std::make_shared<ov::op::v0::Concat>(in, axis);
}

}  // namespace builder
}  // namespace ngraph

File: tools/mo/openvino/tools/mo/front/tf/extractors/concat.py
# Copyright (C) 2018-2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

from openvino.tools.mo.front.common.partial_infer.concat import concat_infer


def tf_concat_ext(pb):
    return {
        'type': 'Concat',
        'N': pb.attr["N"].i,
        'infer': concat_infer
    }
